modules/rosa-hcp-sts-creating-a-cluster-external-auth-provider-delete-cli.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-delete-cli_{context}"]
+= Deleting an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+Delete external authentication providers by using the ROSA CLI.
+
+.Procedure
+
+. Run the following command to display your external authentication provider on your cluster:
```

Comment:
Nit pick: Most of the other procedure steps are in the _To do something, run this command_ format.